check/src/rename.rs

```diff
@@ -323,8 +321,11 @@ pub fn rename(symbols: &mut SymbolModule,
 }
 
 pub fn equivalent(env: &TypeEnv, actual: &ArcType, inferred: &ArcType) -> bool {
+    use substitution::Substitution;
+    // FIXME Inneficient and possible wrong
```

Comment:
Since nothing is actually allocated in `Substitution::new()` and I don't believe there should not be any bugs from having it there I just fixed the spelling. Its not in a hot path of the code so trying to optimize won't have an impact anyway.